### PR TITLE
test(tokenizer): add tests + fix tokenizer

### PR DESCRIPTION
**Pull request by nbsp**
while writing this patch, i found that using `bun test` instead of vitest is much _much_ (up to 100x) faster. i know we don't explicitly target Bun but it might be worth using it in our CI for testing instead of vitest, especially with later integration tests.

### CODE REVIEW

File: agents/src/tokenize/basic/sentence.ts [hunk 1]
```diff
@@ -67,3 +76,7 @@ export const splitSentences = (text: string, minLength = 20): [string, number, n
 
   return sentences;
 };
+
+const rawRegex = (r: RegExp): string => {
```
Comment (lukasIO) [inline agents/src/tokenize/basic/sentence.ts]: wouldn't this be the same as `r.source` ?

Comment (nbsp) [reply agents/src/tokenize/basic/sentence.ts]: i knew there was a thing for this but i couldn't find it for the life of me :woman_facepalming:

File: .changeset/few-schools-run.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@livekit/agents": patch
+---
+
+add tests + fix tokenizer
```
Comment (lukasIO) [inline .changeset/few-schools-run.md]: should the tests bit be part of the changelog?

**Review by lukasIO — APPROVED** [review]

File: .changeset/few-schools-run.md (hunk 2 above)
Comment (nbsp) [reply .changeset/few-schools-run.md]: nope, fixing

**Pull request merged**